modules/core/src/test/java/org/apache/ignite/testframework/junits/IgniteConfigVariationsAbstractTest.java

```diff
@@ -108,9 +92,17 @@ protected static void injectTestsConfiguration(VariationsTestsConfig testsCfgInj
         return false;
     }
 
+    /** Check that test name is not null. */
+    @Before
+    public void checkTestName(){
```

Comment:
What is the purpose of that check? Is it specific and necessary for `IgniteConfigVariationsAbstractTest`?

Reply:
I left it from the previous code. Such check already exists in GridAbstractTest [1].
During run IgniteConfigVariationsAbstractTest subclasses, I did not catch this assertion. But maybe it exists a specific case in which check is necessary.
[1] https://github.com/apache/ignite/blob/master/modules/core/src/test/java/org/apache/ignite/testframework/junits/GridAbstractTest.java#L188